scripts/curate_basel_additional_data.py

```diff
@@ -0,0 +1,56 @@
+import os
+import shutil
+import json
+import argparse
+
+
+def get_parameters():
+    parser = argparse.ArgumentParser(description='Convert dataset to BIDS format.')
+    parser.add_argument("-i", "--path-input",
+                        help="Path to folder containing the dataset to convert to BIDS",
+                        required=True)
+    parser.add_argument("-o", "--path-output",
+                        help="Path to the output BIDS folder",
+                        required=True,
+                        )
+    arguments = parser.parse_args()
+    return arguments
+
+
+def main(path_input, path_output):
+    if os.path.isdir(path_output):
+        shutil.rmtree(path_output)
```

Comment:
This line was probably very helpful for development, but I worry about leaving it in. What if someone accidentally runs this script and it erases their entire home folder?

Reply:
In fact, if you ran this on the pre-existing git repo, how did it avoid just wiping out all the pre-existing data? Did you curate it *then* do an extra step where you copied the curated files into the git repo?